sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -9,34 +9,8 @@
 
 pub mod clients;
 mod generated;
+pub mod helpers;
```

Comment:
The module shouldn't be public, but just export it from the root: https://azure.github.io/azure-sdk/rust_implementation.html#rust-modules

Lots of little modules just makes things hard to find.

Reply:
And if it's not meant to be public, it should be `pub(crate)`: https://azure.github.io/azure-sdk/rust_introduction.html#rust-namespace-naming-internals

Reply:
Hi @heaths, we did consider putting this at the root but felt it was a little better to not have these helpers pollute the root namespace. It just felt a little odd to do `azure_storage_blob::format_page_range` directly. We made the generic enough, so we don't add a bunch more modules later.

If it is truly more idiomatic to export these types of helpers from the root, so be it, just thought I would share my opinion.